pingcap/tiflow
# cdc, pkg: remove capture label in metrics

### What problem does this PR solve?


Issue Number: close #4710 

### What is changed and how it works?


### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Run and see TiCDC dashboard.

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
None
```
